### ISSUE TITLE
Problem with Flash in Nuvola Player 3.0.3 on Kubuntu 14.04

### ISSUE THREAD

**Issue by ZarthKa** [tiliado/nuvolaplayer#241]
Hi there, 
as you can see in header my flash isn't working properly, the weird thing is that it was working in a former install of 14.04 and in 16.04 it was working too. Somehow 16.04 wasn't mine, therefore I decided to reinstall 14.04.

I tried `adobe-flashplugin`,` flashplugin-installer`, and` pepper-flashplugin`, one after another and with the others purged. None of them working.

`hal `is installed, cause I somewhere read that I needed it.

I've read an issue here in this hub , and I provide all the information you needed there. Hope you can help me

`libwebkit2gtk-4.0.37` is installed

The output of a command `ls -l /usr/lib/mozilla/plugins`:

```
lrwxrwxrwx 1 root root 34 Jul  6 15:28 flash-mozilla.so -> /etc/alternatives/flash-mozilla.so
lrwxrwxrwx 1 root root 39 Jul  2 17:16 libjavaplugin.so -> /etc/alternatives/mozilla-javaplugin.so

```

The output of a command `nuvolaplayer3 --debug
`
```
Master: [DEBUG      Gtk] Connecting to session manager
Master: [DEBUG    DioriteGtk] DesktopShell.vala:96: Shell: kwin (null), CSD 0, appmenu 0, menubar 0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Bandcamp at /usr/share/nuvolaplayer3/web_apps/bandcamp, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app TuneIn at /usr/share/nuvolaplayer3/web_apps/tunein, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Yandex Music at /usr/share/nuvolaplayer3/web_apps/yandex_music, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Deezer at /usr/share/nuvolaplayer3/web_apps/deezer, version 2.4
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.15
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Groove at /usr/share/nuvolaplayer3/web_apps/groove, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Jango at /usr/share/nuvolaplayer3/web_apps/jango, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app KEXP Live Stream at /usr/share/nuvolaplayer3/web_apps/kexp, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Logitech Media Server at /usr/share/nuvolaplayer3/web_apps/logitech_media_server, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Mixcloud at /usr/share/nuvolaplayer3/web_apps/mixcloud, version 3.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Plex Music at /usr/share/nuvolaplayer3/web_apps/plex, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app 8tracks at /usr/share/nuvolaplayer3/web_apps/8tracks, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app SoundCloud at /usr/share/nuvolaplayer3/web_apps/soundcloud, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Amazon Cloud Player at /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Spotify at /usr/share/nuvolaplayer3/web_apps/spotify, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Bandcamp at /usr/share/nuvolaplayer3/web_apps/bandcamp, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app TuneIn at /usr/share/nuvolaplayer3/web_apps/tunein, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Yandex Music at /usr/share/nuvolaplayer3/web_apps/yandex_music, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Deezer at /usr/share/nuvolaplayer3/web_apps/deezer, version 2.4
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.15
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Groove at /usr/share/nuvolaplayer3/web_apps/groove, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Jango at /usr/share/nuvolaplayer3/web_apps/jango, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app KEXP Live Stream at /usr/share/nuvolaplayer3/web_apps/kexp, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Logitech Media Server at /usr/share/nuvolaplayer3/web_apps/logitech_media_server, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Mixcloud at /usr/share/nuvolaplayer3/web_apps/mixcloud, version 3.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Plex Music at /usr/share/nuvolaplayer3/web_apps/plex, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app 8tracks at /usr/share/nuvolaplayer3/web_apps/8tracks, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app SoundCloud at /usr/share/nuvolaplayer3/web_apps/soundcloud, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Amazon Cloud Player at /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Spotify at /usr/share/nuvolaplayer3/web_apps/spotify, version 2.1
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_amazon_cloud_player 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_jango 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_kexp 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_logitech_media_server 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_mixcloud 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_plex 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_spotify 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_test 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_tunein 48 not found.
Master: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_yandex_music 48 not found.
Master: [DEBUG    DioriteGtk] application.vala:219: XFCE session vanished: org.xfce.SessionManager
```

When I start Amazon Cloud Player

```
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Bandcamp at /usr/share/nuvolaplayer3/web_apps/bandcamp, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app TuneIn at /usr/share/nuvolaplayer3/web_apps/tunein, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Yandex Music at /usr/share/nuvolaplayer3/web_apps/yandex_music, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Deezer at /usr/share/nuvolaplayer3/web_apps/deezer, version 2.4
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.15
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Groove at /usr/share/nuvolaplayer3/web_apps/groove, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Jango at /usr/share/nuvolaplayer3/web_apps/jango, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app KEXP Live Stream at /usr/share/nuvolaplayer3/web_apps/kexp, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Logitech Media Server at /usr/share/nuvolaplayer3/web_apps/logitech_media_server, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Mixcloud at /usr/share/nuvolaplayer3/web_apps/mixcloud, version 3.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Plex Music at /usr/share/nuvolaplayer3/web_apps/plex, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app 8tracks at /usr/share/nuvolaplayer3/web_apps/8tracks, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app SoundCloud at /usr/share/nuvolaplayer3/web_apps/soundcloud, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Amazon Cloud Player at /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Spotify at /usr/share/nuvolaplayer3/web_apps/spotify, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Bandcamp at /usr/share/nuvolaplayer3/web_apps/bandcamp, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app TuneIn at /usr/share/nuvolaplayer3/web_apps/tunein, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Yandex Music at /usr/share/nuvolaplayer3/web_apps/yandex_music, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Deezer at /usr/share/nuvolaplayer3/web_apps/deezer, version 2.4
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Google Play Music at /usr/share/nuvolaplayer3/web_apps/google_play_music, version 5.15
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Groove at /usr/share/nuvolaplayer3/web_apps/groove, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Test at /usr/share/nuvolaplayer3/web_apps/test, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Jango at /usr/share/nuvolaplayer3/web_apps/jango, version 2.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app KEXP Live Stream at /usr/share/nuvolaplayer3/web_apps/kexp, version 1.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Logitech Media Server at /usr/share/nuvolaplayer3/web_apps/logitech_media_server, version 2.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Mixcloud at /usr/share/nuvolaplayer3/web_apps/mixcloud, version 3.0
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Plex Music at /usr/share/nuvolaplayer3/web_apps/plex, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app 8tracks at /usr/share/nuvolaplayer3/web_apps/8tracks, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app SoundCloud at /usr/share/nuvolaplayer3/web_apps/soundcloud, version 1.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Amazon Cloud Player at /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player, version 5.1
Master: [DEBUG    Nuvola] WebAppRegistry.vala:169: Found web app Spotify at /usr/share/nuvolaplayer3/web_apps/spotify, version 2.1
Master: [INFO     Nuvola] WebAppRegistry.vala:89: Using web app Amazon Cloud Player, version 5.1, data dir /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player
Master: [DEBUG    Nuvola] MasterController.vala:463: Launch app runner for 'amazon_cloud_player': /usr/lib/nuvolaplayer3/apprunner -D -a /usr/share/nuvolaplayer3/web_apps/amazon_cloud_player 
Runner: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_amazon_cloud_player 1024 not found.
Runner: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_amazon_cloud_player 1024 not found.
Runner: [DEBUG      Gtk] Connecting to session manager
Runner: [DEBUG    DioriteGtk] DesktopShell.vala:96: Shell: kwin (null), CSD 0, appmenu 0, menubar 0
Runner: [DEBUG    DioriteGtk] application.vala:219: XFCE session vanished: org.xfce.SessionManager
Master: [DEBUG    Nuvola] MasterController.vala:287: Connected to runner server for 'amazon_cloud_player'.
Runner: [DEBUG    Nuvola] Connection.vala:132: Network Proxy: system settings
Runner: [DEBUG    Nuvola] WebEngine.vala:73: Nuvola WebKit Extension directory: /usr/lib/nuvolaplayer3
Runner: [WARNING  GLib-GObject] g_object_set_property: construct property "local-storage-directory" for object 'WebKitWebContext' can't be set after construction
Runner: [DEBUG    Nuvola] WebApp.vala:209: Theme icon nuvolaplayer3_amazon_cloud_player 20 not found.
Worker: [DEBUG    Nuvola] WebExtension.vala:220: Page 1 created for (null)
Runner: [DEBUG    Nuvola] Notifications.vala:175: Notifications: persistence false, actions true, icons false
Master: [DEBUG    Nuvola] MediaKeys.vala:130: GNOME settings daemon vanished: org.gnome.SettingsDaemon
Master: [DEBUG    Nuvola] XKeyGrabber.vala:77: Grabbed XF86AudioPrev, count 1
Master: [DEBUG    Nuvola] XKeyGrabber.vala:77: Grabbed XF86AudioNext, count 1
Master: [DEBUG    Nuvola] XKeyGrabber.vala:77: Grabbed XF86AudioPlay, count 1
Master: [DEBUG    Nuvola] XKeyGrabber.vala:77: Grabbed XF86AudioStop, count 1
Master: [DEBUG    Nuvola] XKeyGrabber.vala:77: Grabbed XF86AudioPause, count 1
Worker: [DEBUG    Nuvola] WebExtension.vala:291: Document loaded about:loading
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component tray_icon (Tray Icon) enabled
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component unity_launcher (Unity Laucher) enabled
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component notifications (Notifications) enabled
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component scrobbler (Audio Scrobbler Services) enabled
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component mpris (MPRIS 2) enabled
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component lyrics (Lyrics) enabled
Runner: [DEBUG    Nuvola] AppRunnerController.vala:451: Component developer (Developer's tools) enabled
Runner: [DEBUG    Nuvola] MPRISProvider.vala:73: Bus acquired: org.mpris.MediaPlayer2.NuvolaAppAmazonCloudPlayer, registering objects
Runner: [DEBUG    Nuvola] MPRISProvider.vala:90: Bus name acquired: org.mpris.MediaPlayer2.NuvolaAppAmazonCloudPlayer
Runner: [DEBUG    Nuvola] WebEngine.vala:498: Navigation, current window: uri = https://music.amazon.de/new, result = false, frame = (null), type = WEBKIT_NAVIGATION_TYPE_OTHER
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Trying to play /usr/share/nuvolaplayer3/audio/audiotest.mp3.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from NULL to READY.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from READY to PAUSED.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: Pipeline state changed from PAUSED to PLAYING.
Runner: [DEBUG    Nuvola] FormatSupport.vala:103: End of stream for file /usr/share/nuvolaplayer3/audio/audiotest.mp3.
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Shockwave Flash (enabled, flash) at /usr/lib/mozilla/plugins/flash-mozilla.so: Shockwave Flash 13.1 r2
Runner: [DEBUG    Nuvola] FormatSupportCheck.vala:145: Nuvola.WebPlugin: Java(TM) Plug-in 11.91.2 (enabled, not flash) at /usr/lib/mozilla/plugins/libjavaplugin.so: Next Generation Java Plug-in 11.91.2 for Mozilla browsers
Runner: [DEBUG    Nuvola] WebAppWindow.vala:170: Sidebar position: 960
Runner: [INFO     Nuvola] AppRunnerController.vala:236: Logged in as  ()
Worker: [DEBUG    Nuvola] WebExtension.vala:130: Window object cleared for 'https://music.amazon.de/new'
Worker: [DEBUG    Nuvola] WebExtension.vala:140: Init frame: https://music.amazon.de/new, 0x1d6e220, 0x1bd8390, 0x7f9b8013f4b0
Runner: [fresh] [error] probe_ppp_module, can't find libpepflashplayer.so
Runner: [fresh] [error] probe_ppp_module, can't find libpepflashplayer.so
[fresh] [error] probe_ppp_module, can't find libpepflashplayer.so
Worker: [DEBUG    Nuvola] WebExtension.vala:291: Document loaded https://music.amazon.de/new
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN rxstores not found
Runner: [DEBUG    Nuvola] DeveloperSidebar.vala:158: Media player notify: artwork-location
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/2016527224930332/cloudPlayerWeb_de_DE._CB268691260_.js:12:2254: CONSOLE WARN Deprecation warning: moment().subtract(period, number) is deprecated. Please use moment().subtract(number, period).
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Prop filter of Card is not type string
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Prop filter of Card is not type string
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Prop filter of Card is not type string
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Prop filter of Card is not type string
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Prop filter of Card is not type string
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:2:23501: CONSOLE WARN Cannot compute playbackState, nowPlaying is not available

```

And when I try to play music

```
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256
Runner: https://images-eu.ssl-images-amazon.com/images/G/01/digital/music/player/web/assets/20165282451346/digitalMusicWebPlayer._CB268677898_.js:30:28341: CONSOLE LOG 256 256


```

**Comment by ZarthKa** [comment]
Newsflash: I'm totally overwhelmed by myself. True story: I got it working. Here's what I've made wrong before:

I dismissed the two steps when you install pepperflashplugin-nonfree:
```
sudo apt-get install pepperflashplugin-nonfree
sudo update-pepperflashplugin-nonfree --install
```
To be true, I'm not shure if I dismissed them, they didn't work because google did something with their signing key found [here](https://wiki.debian.org/PepperFlashPlayer).

I made a`
gpg --keyserver pgp.mit.edu --recv-keys 1397BC53640DB551 gpg --export --armor 1397BC53640DB551  sudo sh -c 'cat >> /usr/lib/pepperflashplugin-nonfree/pubkey-google.txt'`

Then I played around with nuvolas settings → format support, and there it was: **sound**!!!
